src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="big_searchbar"
+        type="search"
+        name="bigSearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+
+      <img
+      class="icon"
+      src="images/Search.svg"
+      alt="search"
+      >
```

Comment:
fix aatributes formatting, [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 5th item, fix it everywhere not only here
```suggestion
      <img
        class="icon"
        src="images/Search.svg"
        alt="search"
      >
```

Reply:
ty for review and advices. i've already fixed issues.